scripts/igmp_query_injector.py

```diff
@@ -410,6 +139,10 @@ def network_backend_is_openvswitch():
 
 
 def igmp_snooping_is_enabled_on_bridge(bridge):
```

Comment:
If it is too expensive to call this function for lots of vifs, you can memoize it: http://code.activestate.com/recipes/578231-probably-the-fastest-memoization-decorator-in-the-/